### PR TITLE
Minimum Sum Partition in C++

### PR DESCRIPTION
**Pull request by sharur7**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

HackinCodes2020 Participant

In this problem given set is partitioned into two subsets such that the difference between the sum of elements of 1st set and the sum of elements of 2nd set is minimized.

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

### Related Issues or Pull Requests

#805

### CODE REVIEW

**Comment by sharur7** [comment]
Please review this PR @SKAUL05 @iamrajiv @Kajol-Kumari @plazzy99

**Review by iamrajiv — CHANGES_REQUESTED** [review]
@sharur7 Update the README. Rename the program like this `Minimum_Sum_Partition.cpp `.

**Comment by sharur7** [comment]
> @sharur7 Update the README. Rename the program like this `Minimum_Sum_Partition.cpp `.

@iamrajiv Done!

File: C-Plus-Plus/dp/Minimum_Sum_Partition.cpp [hunk 1]
```diff
@@ -0,0 +1,75 @@
+// Minimum Sum Partition
+#include <iostream>
+#include <string>
+using namespace std;
+
+// Partition the Set into two subsets Set1, Set2 such that the
```
Comment (iamrajiv) [inline C-Plus-Plus/dp/Minimum_Sum_Partition.cpp]: @sharur7 Convert all single-line comments into one multiline comment. Something like this
```
/* 
...
*/
```

**Review by iamrajiv — CHANGES_REQUESTED** [review]

File: C-Plus-Plus/dp/Minimum_Sum_Partition.cpp (hunk 1 above)
Comment (sharur7) [reply C-Plus-Plus/dp/Minimum_Sum_Partition.cpp]: Done @iamrajiv

**Review by iamrajiv — APPROVED** [review]

**Pull request merged**